cli/src/util/spm.ts

```diff
@@ -26,17 +34,69 @@ export async function findPackageSwiftFile(config: Config): Promise<string> {
   return resolve(packageDirectory, 'Package.swift');
 }
 
-export async function generatePackageFile(config: Config, plugins: Plugin[]): Promise<void> {
+export async function generatePackageSwiftFile(config: Config, plugins: Plugin[]): Promise<void> {
```

Comment:
exported functions can be used from javascript, so we can't rename it as it would be breaking if anybody is using them.